File: ProjetoProgSofApp/php/src/model/Cliente.php
<?php

namespace model;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'tb_cliente')]
class Cliente extends GenericModel
{
    #[ORM\Column(type: 'string', name: 'nome_cliente')]
    private $nome;

    #[ORM\Column(type: 'string', name: 'email_cliente', unique: true)]
    private $email;

    #[ORM\Column(type: 'string', name: 'cpf_cliente', length: 11, unique: true, nullable: true)]
    private $cpf;

    #[ORM\Column(type: 'string', name: 'senha_cliente', nullable: true)]
    private $senha;

    #[ORM\Column(type: 'boolean', name: 'is_admin', options: ['default' => false])]
    private bool $isAdmin = false;

    public function getNome()
    {
        return $this->nome;
    }

    public function setNome($nome): void
    {
        $this->nome = $nome;
    }

    public function getEmail()
    {
        return $this->email;
    }

    public function setEmail($email): void
    {
        $this->email = $email;
    }

    public function getCpf()
    {
        return $this->cpf;
    }

    public function setCpf($cpf): void
    {
        $this->cpf = $cpf;
    }

    public function getSenha()
    {
        return $this->senha;
    }

    public function setSenha($senha): void
    {
        $this->senha = $senha;
    }

    public function getIsAdmin(): bool
    {
        return $this->isAdmin;
    }

    public function setIsAdmin(bool $isAdmin): void
    {
        $this->isAdmin = $isAdmin;
    }
}

File: src/main/java/br/edu/ifpr/bsi/ProjetoProgSofApp/models/Cliente.java
package br.edu.ifpr.bsi.ProjetoProgSofApp.models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name = "tb_cliente")
public class Cliente extends GenericModel {

    @Column(name = "nome_cliente", nullable = false)
    private String nome;

    @Column(name = "email_cliente", unique = true, nullable = false)
    private String email;

    @Column(name = "cpf_cliente", length = 11, unique = true)
    private String cpf;

    @Column(name = "senha_cliente")
    private String senha;

    @Column(name = "is_admin")
    private Boolean isAdmin = false;
}